packages/create-sitecore-jss/src/templates/nextjs-styleguide/scripts/disconnected-mode-proxy.ts

```diff
@@ -12,17 +12,17 @@
 import fs from 'fs';
 import path from 'path';
 import { createDefaultDisconnectedServer } from '@sitecore-jss/sitecore-jss-dev-tools';
-import { config } from '../package.json';
+import config from '../src/temp/config';
 
 const touchToReloadFilePath = 'src/temp/config.js';
 
 const serverOptions = {
   appRoot: path.join(__dirname, '..'),
-  appName: config.appName,
+  appName: config.jssAppName,
   // Prevent require of ./sitecore/definitions/config.js, because ts-node is running
   requireArg: null,
   watchPaths: ['./data'],
-  language: config.language,
+  language: config.defaultLanguage,
```

Comment:
We don't have `defaultLanguage` (yet) in Next.js config generation. See previous comment.

Reply:
Good catch, I forgot that we don't generate `defaultLanguage` in the config